subsys/bluetooth/host/Kconfig

```diff
@@ -761,6 +761,21 @@ config BT_SCAN_WITH_IDENTITY
 	  for for the local identity. If this use case is required, then enable
 	  this option.
 
+config BT_SCAN_AND_INITIATE_IN_PARALLEL_ALLOWED
```

Comment:
Open question: Should there be a separate Kconfig entry for this functionality, or should we support concurrent scanning and initiating when the prerequisites are met?

Reply:
I like the kconfig as it makes the dependencies apparent.
I would drop the `_ALLOWED` from the name.